utils.py

```diff
@@ -28,19 +33,20 @@ def get_scan_area():
     area = int(width * height)
     return area
 
+def get_scan_area():
+    if isinstance(config.GRID[0], (int, long, float, complex)):
```

Comment:
Again - better convert config.GRID to list early and have a single version of this function.